client/src/components/settings/SettingsList.vue

```diff
@@ -0,0 +1,187 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="settings-list-container">
+    <ion-list class="settings-list">
+      <!-- change lang -->
+      <settings-option
+        :title="'SettingsModal.language.label'"
+        :description="'SettingsModal.language.description'"
+      >
+        <ms-dropdown
+          class="dropdown"
+          :options="languageOptions"
+          :default-option-key="I18n.getLocale()"
+          @change="changeLang($event.option.key)"
+        />
+      </settings-option>
+      <!-- change theme -->
+      <!-- TODO: REMOVE "disabled=true' : " WHEN DARK MODE WILL BE HERE: https://github.com/Scille/parsec-cloud/issues/5427 -->
+      <settings-option
+        :title="'SettingsModal.theme.label'"
+        :description="'SettingsModal.theme.description'"
+      >
+        <ms-dropdown
+          class="dropdown"
+          :options="themeOptions"
+          :default-option-key="config.theme"
+          @change="changeTheme($event.option.key)"
+          :disabled="true"
+        />
+      </settings-option>
+      <!-- minimize in status bar -->
+      <settings-option
+        v-if="isPlatform('electron') && !isMacOS()"
+        :title="'SettingsModal.minimizeToSystemTray.label'"
+        :description="'SettingsModal.minimizeToSystemTray.description'"
+      >
+        <ion-toggle v-model="config.minimizeToTray" />
+      </settings-option>
+      <!-- MacOS-only toggle confirm -->
+      <settings-option
+        v-if="isPlatform('electron') && isMacOS()"
+        :title="'SettingsModal.confirmBeforeQuit.label'"
+        :description="'SettingsModal.confirmBeforeQuit.description'"
+      >
+        <ion-toggle v-model="config.confirmBeforeQuit" />
+      </settings-option>
+      <!-- send error report -->
+      <settings-option
+        :title="'SettingsModal.enableTelemetry.label'"
+        :description="'SettingsModal.enableTelemetry.description'"
+      >
+        <ion-toggle v-model="config.enableTelemetry" />
+      </settings-option>
+    </ion-list>
+
+    <ion-list class="settings-list">
+      <!-- open config dir -->
+      <settings-option
+        title="SettingsModal.configDir.label"
+        description="SettingsModal.configDir.description"
+      >
+        <ion-button @click="openConfigDir">
+          {{ $msTranslate('SettingsModal.configDir.open') }}
+        </ion-button>
+      </settings-option>
+
+      <settings-option
```

Comment:
Nop, not what it looks like on master: https://github.com/Scille/parsec-cloud/blob/7739f08858c8c4fd3d8f4a2446c2783eb19c24bd/client/src/views/settings/SettingsModal.vue#L121